apiserver/facades/client/modelgeneration/modelgeneration.go

```diff
@@ -276,6 +289,72 @@ func (api *API) BranchInfo(args params.BranchInfoArgs) (params.GenerationResults
 	return result, nil
 }
 
+// ShowCommit will return details a commit given by its generationId
+// An error is returned if either no branch can be found corresponding to the generation id.
+// Or the generation id given is below 1.
+func (api *APIV3) ShowCommit(arg params.GenerationId) (params.GenerationCommitResult, error) {
```

Comment:
```suggestion
func (api *API) ShowCommit(arg params.GenerationId) (params.GenerationCommitResult, error) {
```

API is actually APIV4 and you end up masking it.